cmd/minishift/cmd/start.go

```diff
@@ -221,16 +223,29 @@ func handleProxies() *util.ProxyConfig {
 	return proxyConfig
 }
 
+func getSlice(key string) []string {
+	if viper.IsSet(key) {
+		return viper.GetStringSlice(key)
+	}
+	return nil
+}
+
+func determineInsecureRegistry(key string) []string {
+	s := getSlice(key)
+	return append(s, defaultInsecureRegistry...)
```

Comment:
This will not de-dup though, right? If the user specified "172.30.0.0/16" himself, it will end up twice in the final slice.